atxsdata/data.go

```diff
@@ -204,6 +215,33 @@ func (d *Data) IterateInEpoch(epoch types.EpochID, fn func(types.ATXID, *ATX), f
 	}
 }
 
+func (d *Data) IterateHighTicksInEpoch(target types.EpochID, fn func(types.ATXID) bool, filters ...AtxFilter) {
```

Comment:
Is `filters` argument ignored in this function?

Reply:
Removed it. The `IterateHighTicksInEpoch` already filters out malicious ATXs